.github/workflows/ci-build.yml

```diff
@@ -145,3 +145,15 @@ jobs:
             oft
       - name: Run OpenFastTrace
         run: ./ci/trace-requirements.sh
+
+  go-dependency-check:
```

Comment:
Is there a specific reason why you moved this check from `dependencies_check.yml` to `ci-build.yml`?

Reply:
dependencies_check.yml contains the java dependency check, this is just for go dependencies.